projects/elife.yaml

```diff
@@ -363,3 +371,48 @@ search:
         ports:
             - 22
             - 80
+
+bus:
+    default-branch: null
+    domain: null
+    int_domain: null
+    aws:
+        ec2: false
+        sns:
+            # bus-articles-prod, bus-articles-end2end, etc.
+            - bus-articles--{cluster}
```

Comment:
I think we're on the same page here, just our vocabulary is unclear.

I'm going to go with `instance` and update any references to `cluster` that may have slipped in.